src/components/NormalRightRow/index.jsx

```diff
@@ -1,19 +1,38 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import { shape, string } from 'prop-types';
+import Typography from '@material-ui/core/Typography';
 
 function NormalRightRow({ schema, classes }) {
-  const keywords = Object.keys(schema);
+  const skipKeywords = ['type', 'name', 'description', 'items', 'contains'];
+  const keywords = Object.keys(schema).filter(
+    key => !skipKeywords.includes(key)
+  );
 
   return (
     <div className={`${classes.row} ${classes.rightRow}`}>
       <div className={classes.keywordColumn}>
-        {keywords.map(keyword => (
-          <p key={keyword} className={classes.line}>
-            {keyword}
-            {': '}
-            {schema[keyword]}
-          </p>
-        ))}
+        {keywords.length === 0 ? (
+          <Typography
+            component="div"
+            variant="subtitle2"
+            className={classes.line}>
+            <br />
+          </Typography>
+        ) : (
+          <Fragment>
+            {keywords.map(keyword => (
+              <Typography
+                key={keyword}
+                component="div"
+                variant="subtitle2"
+                className={classes.line}>
+                {keyword}
+                {': '}
+                {`${schema[keyword]}`}
+              </Typography>
+            ))}
+          </Fragment>
```

Comment:
I think we can render without `<Fragment /> here.

Reply:
Yep, thanks! Applied changes!